charts/mysql-operator/values.yaml

```diff
@@ -14,8 +14,8 @@ nameOverride: ""
 fullnameOverride: ""
 
 manager:
-  image: zhyass/manager
-  tag: 0.1
+  image: acekingke/controller
```

Comment:
Don't modify temporarily, we'll build and push the image to xenondb or radondb org in docker hub.